app/views/green_lanes/results/_result_category_3.html.erb

```diff
@@ -64,4 +64,6 @@
 
 <%= render 'green_lanes/shared/about_your_goods_card', title: 'Your movement of goods', edit_path: 'EDIT GOOD DETAILS PAGE' %>
 
-<%= render_exemptions(@assessments, @category) %>
+<% unless @answers.nil? %>
```

Comment:
If the user is cat 3 and hasnt been shown any answers then we dont show the below sections